content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,297 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a <abbr>model registry</abbr> to manage the lifecycle of the models
+we get from our experiments. Using DVC and DVC [Studio](/doc/studio) we will set
+up a model registry where we can discover, share, deploy and audit all our
+models and which will serve as the single source of truth for our model
+management.
+
+You can also set up the model registry locally without Studio, using a command
+line tool called [Git Tag Ops (GTO)](<(/doc/gto)>). We will see how this is done
+[later in this chapter](#GTO-tip).
+
+<details id="follow-along-instructions">
+
+#### 💡 Expand to see how to set things up to follow along with the guide
+
+You can fork our
+[example repository](https://github.com/iterative/example-get-started-experiments)
```

Comment:
```suggestion
[example repository](https://github.com/iterative/example-get-started-experiments/fork)
```

Minor: we might as well point them directly to the fork. I worry otherwise people might miss it and clone it directly.

Reply:
Maybe we should even add a suggestion to fork into https://github.com/iterative/example-get-started-experiments#readme?